7-ExecutablesAndLibraries/README.md

```diff
@@ -1 +1,42 @@
-# Executables and libraries
\ No newline at end of file
+# Executables and libraries
+
+This chapter explains how to create executables and libraries. Executables and libraries are essential for managing projects
+with CMake.
+
+Executables and libraries are targets. A target is (just as in a Makefile) something that is going to be built, and which can be a required dependency
+for another target.
+
+# Executable
+
+An executable target is used to create for example a `.exe` file or for cross platform a `.elf`, `.hex`, or a `.elf`file.
+
+In CMake, an executable target is created using the following command.
+
+```CMake
+add_executable(target_name [sources])
+```
+
+The target name is required, whereas the sources are optional and can be specified later.
+
+# Libraries
+
+A library can be used to reuse compiled source files for different other libraries or executable.
```

Comment:
"different other" the end of the line reads awkwardly

Reply:
Fixed